web/src/features/moreCast2/components/GridComponentRenderer.tsx

```diff
@@ -104,20 +107,90 @@ export class GridComponentRenderer {
     // We can disable a cell if an Actual exists or the forDate is before today.
     // Both forDate and today are currently in the system's time zone
     const isPreviousDate = isPreviousToToday(params.row['forDate'])
-
     const isGrassField = field.includes('grass')
+    const label = isGrassField || isPreviousDate ? '' : createWeatherModelLabel(params.row[field].choice)
+    const formattedValue = parseFloat(params.formattedValue)
+    const actualField = this.getActualField(field)
+    const actualValue = params.row[actualField]
+    let showLessThan = false
+    let showGreaterThan = false
+    // Only show + and - icons if an actual value exists, a forecast value exists and this is not a windDirection
+    // field.
+    if (!isNaN(actualValue) && isNumber(actualValue) && isNumber(formattedValue) && !field.includes('windDirection')) {
+      showLessThan = formattedValue < actualValue
+      showGreaterThan = params.formattedValue > actualValue
+    }
+
+    // The grass curing 'forecast' field is rendered differently
+    if (isGrassField) {
```

Comment:
I think we can pull this out into a component and get some test coverage around it

Reply:
Good call. I pulled out a ForecastCell component and created some tests around it.